crates/bevy_reflect/derive/src/from_reflect.rs

```diff
@@ -41,10 +41,11 @@ pub(crate) fn impl_enum(reflect_enum: &ReflectEnum) -> proc_macro2::TokenStream
     let bevy_reflect_path = reflect_enum.meta().bevy_reflect_path();
 
     let ref_value = Ident::new("__param0", Span::call_site());
-    let EnumVariantConstructors {
+    let EnumVariantOutputData {
```

Comment:
it looks strange to have `..`; the rest of the struct is due to some implementation detail and is never used.

Maybe the functions should just output `(variant_names, variant_constructors)`

Reply:
Oops, the `..` is leftover from a previous commit where I also output a `variant_patterns` field. I removed it because it wasn't used yet. I actually do end up adding it back in a PR that I'm going to post soon based on this one.

Do you think the tuple is still clearer if in that future PR I only make use of `variant_constructors` and `variant_patterns`?

Ideally, I'd have a way to create each of the three fields conditionally, but I don't know how to do that in a clean way without either wrapping them in `Option`, defaulting them to an empty `Vec`, or iterating through all variant fields twice.

Reply:
Ah I misread and though the output was `EnumVarianOutputDataBuilder`; it it's just `EnumVariantOutputData` I think it's fine